### ISSUE TITLE
Error on trabslate

### ISSUE THREAD

**Issue by lrntmszrs** [johannschopplich/kirby-content-translator#12]
Hi, I encounter an error when I click on the translate button. I've try on a plain kit clean install, with two languages and a simple `textarea` field… Sync works, but when I click _translate_ I get an error message.

In the console tab, of the browser, a message appears when I click

```
index.js?1714369334:1

TypeError: Cannot read properties of undefined (reading 'text')
    at index.js?1714369334:1:2393
    at async Promise.all (:8888/deepl/panel/pages/index 0)
    at async I (index.js?1714369334:1:2982)
    at async recursiveTranslateContent (index.js?1714369334:1:2850)
    at async Object.H (index.js?1714369334:1:5466)

```

The message error is not very helpful, so I'm quite stuck…

![image](https://github.com/johannschopplich/kirby-content-translator/assets/36987053/ae7950a5-0eba-4092-b400-53db7a9dbfff)

Any Idea ?
Thanks 

Lrt

--
Kirby 4.2.0
Php 8.1.13

**Comment by johannschopplich** [comment]
Seems like your server-side request to the DeepL API failed and thus the response in the Kirby Panel is empty. Can you please provide a minimum reproducible example (a repo) so I can test your issue? Thanks!

**Comment by johannschopplich** [comment]
Sounds good. Thanks!

Can you show me the the response log from your browser's console? Specifically, the request error from the response body. Please enable Kirby debug mode therefore.

**Comment by lrntmszrs** [comment]
My browser displays these 2 errors in the console panel... (I'm not sure if this meets your request?)
![image](https://github.com/johannschopplich/kirby-content-translator/assets/36987053/d7d6f362-8078-4de5-9fe9-c34176660d5d)

**Comment by johannschopplich** [comment]
No, I mean the result of the fetch request happening before the JS error.

**Comment by lrntmszrs** [comment]
Oops… sorry. I'm afraid, I don't really know where to look…

**Comment by johannschopplich** [comment]
You can look in your browser's network tab. There is probably a red item, indicating the failed request. The response is important, because the DeepL translation is happening on the server-side, not client-side (Kirby Panel).

**Comment by lrntmszrs** [comment]
wow thanks ! I'd never used this tab before.

**Request header**
![image](https://github.com/johannschopplich/kirby-content-translator/assets/36987053/e651d90b-5328-4c39-bd71-199b31dcb345)

**Response headers**
![image](https://github.com/johannschopplich/kirby-content-translator/assets/36987053/d37abe36-caac-4741-894f-792e7dc50689)

The preview shows that the right information is being sent... and received. It seems that Kirby is unable to update the page...

**Request preview**
![image](https://github.com/johannschopplich/kirby-content-translator/assets/36987053/b4554892-1d76-4a2e-b442-63a6240e1526)
**Response preview**
![image](https://github.com/johannschopplich/kirby-content-translator/assets/36987053/b839faed-29a2-4c91-913c-483bb2f433b8)

**Comment by johannschopplich** [comment]
@lrntmszrs This issue should be resolved with the latest build. Please follow my instructions above. Thanks!

**Comment by lrntmszrs** [comment]
Absolutely, everything works fine now. I had indeed DL the `main` branch
Thank you very much for your help !
